docs/README-EDITING.md

```diff
@@ -66,77 +51,120 @@ sitemap:
 
-### Live demo:
-It is able to run code snippets to show result of the code:
+A [`<semver.version>` directory](./README.md#handsontable-docs-directory-structure) with the largest version number gets tagged as the `latest` version of the docs.
 
-```
-    ::: example #example1
+To display the latest docs version in a browser:
+1. Start a [local Handsontable docs server](./README.md#getting-started-with-handsontable-docs).
+2. In your browser, go to http://localhost:8080/docs.
 
-    ```js
-        // code here
-    ```
+### Editing an archived docs version
 
-    :::
-```
+To edit an archived docs version, go to the required [`<semver.version>` directory](./README.md#handsontable-docs-directory-structure).
 
-`#example1` is id for creating handsontable container.
- 
-### Editing the `Next` version
+To display an archived docs version in a browser:
+1. Start a [local Handsontable docs server](./README.md#getting-started-with-handsontable-docs).
+2. In your browser, go to http://localhost:8080/docs/{semver.version}/.
 
-*The next version is only available on the `localhost` and on the staging server.*
+## Editing the API reference
 
-Files included in next versions live in dir `/next/`.
-To display it in a browser, please go into URL: `/docs/next`.
+The API reference is generated automatically from the source code.
 
-### Editing the `latest` version
+The API reference output lives in the `/next/api/` directory.
 
-The latest version is the largest semver valid directory inside the root directory. 
-It is in `/[number]/`.
+To edit the API reference, go into the source code and change the JSDoc comments.
 
-To display the latest docs version in a browser, please go into: `/docs/`.
+To generate the API reference output, run the following command from the `/docs/` directory:
 
-### Editing an archival version
+```bash
+npm run docs:api
+```
 
-Directory: `/[number]/`.
+### JSDoc links
 
-In a browser please go into: `/docs/[number]`.
+As we want to support case-insensitive URLs, follow these guidelines when editing the JSDoc links:
 
-## Editing API Reference
+* Use lower case characters.
+* To separate words before `#`, use `-`.
 
-To edit API Reference goes into source code and change jsdocs comments. Then, to generate API Reference pages:
+```js
+// Wrong:
+/** {@link Options#autoColumnSize}. */
 
-```shell script
-npm run docs:api
+// OK: 
+/** {@link options#autocolumnsize Options#autoColumnSize}. */
+
+
+// Wrong:
+/** {@link HiddenRows} */
+
+// OK:
+/** {@link hidden-rows HiddenRows} */
 ```
 
-## Versioning
+## Adding code examples
+Inside the Markdown files, you can add code snippets that will show the code's result:
+
+```js
+    ::: example #example1
+    // `#example1` is the example's ID, for creating a Handsontable container
 
-**Release a version**
+    ```js
+        // code here
+    ```
 
-```shell script
-npm run docs:version <semver_number>
-# for instance 
-# npm run docs:version 9.0
+    :::
```

Comment:
If could I ask you, please describe options for this md container

```
::: example #exampleId .class :preset --html 1 --js 2 --css 3  --no-edit --tab preview
```

* #example - container id 
* .class - container custom class 
* :preset - determine which dependencies should be loaded 

https://github.com/handsontable/handsontable/blob/c505c718ae9047a72ef9a970c7c2cf8d6b5369d8/docs/.vuepress/handsontable-manager/dependencies.js#L75-L91

* --html <pos>  --js <pos> --css <pos>  : determine positions of code snippets 

* --no-edit - do not add an `edit` button
* --tab - determine which tab is open as default, values:  (code|html|css|preview)

Reply:
@Siemienik sure, check out the draft:

https://github.com/handsontable/handsontable/pull/8248/commits/74aff192e1bf94ce45990b19e6ff374e76b0f644